src/pages/TripDetailPage.tsx

```
@@ -0,0 +1,53 @@
+import AppBar from '@/components/common/AppBar';
+import FullPageLoading from '@/components/common/FullPageLoading';
+import Map from '@/components/common/Map';
```

Biome (2.1.2)

[error] Do not shadow the global "Map" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)